.gitignore

```diff
@@ -41,6 +41,7 @@ atlassian-ide-plugin.xml
 .idea/compiler.xml
 .idea/gradle.xml
 .idea/jarRepositories.xml
+.idea/scala_settings.xml
```

Comment:
This is Scala settings that IntelliJ decided to add to record-layer for some reasons. I am not sure why it does so, but since we don't use Scala here, I thought it is ok to ignore local changes to it by ignoring the file itself.